package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "widget",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "preview": "vite preview",
+    "prettier": "./node_modules/.bin/prettier . --write",
```

Comment:
Do you really need this path to Prettier in node_modules? Looks strange, why not just `yarn run prettier`?